src/compute_changes.rs

```diff
@@ -370,28 +377,33 @@ pub fn compute_changeset(
 
     let changes = matches
         .into_iter()
-        .map(|x| {
-            let new_delta = match &x.delta {
+        .map(|ChangedTask{ orig, delta }| {
```

Comment:
Oh, and this is missing a space between `ChangedTask` and `{`, according to `cargo fmt`.